Clients/src/App.tsx

```diff
@@ -22,7 +22,7 @@ function App() {
       <CssBaseline />
       <Routes>
         <Route path="/" element={<Dashboard />}>
-          <Route path="/" element={<Home />} />
+          <Route path="/" element={<Home projects={projects} />} />
```

Comment:
Thank you for your try on applying changes. 🙏🏻
This file, `App.tsx` is the root file for our routing system. No data should be passed to elements. Anything like this 
`projectData` that holds mock data, should be passed inside child elements. **At least for now.**
Remember that we can always jump on a call, so I can help with questions you may have. ✌️